spec/models/user_spec.rb

```diff
@@ -55,4 +55,18 @@
       expect(user1).to be false
     end
   end
+
+  describe 'review notification' do
+    let(:user) { create(:user) }
+    before do
+      Timecop.return_to_baseline
+      @card = user.cards.create(original_text: 'hello', translated_text: 'Привет',
+                                review_date: (DateTime.current + 1.day))
+    end
+    it 'change deliveries count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.